### DISCUSSION TITLE
I tried to run nfo --help but I don't know how to do it properly and just getting errors

### DISCUSSION THREAD

**Discussion by djdancer** [rlaphoenix/pynfogen#12] [Q&A] [answered]
Can I ask for a help how to run the command properly? Maybe a video tutorial would help.? Thank you very much in advance.

![WindowsTerminal_KEnyXSKcLD](https://github.com/rlaphoenix/pynfogen/assets/54954165/17488c66-ce6a-47e2-a164-193459d00da8)

**Comment by rlaphoenix** [comment]
Hi, you are trying to run the executable shim `nfo` from *within* the Python shell. You need to exit it and run it from outside Python, in a normal terminal environment.

**Reply by djdancer** [reply]
Thanks for your answer! I tried your suggestion but I getting these errors. Do you have any advise how could I repair/fix it?

![WindowsTerminal_EAhHgtX7kG](https://github.com/rlaphoenix/pynfogen/assets/54954165/00052096-b5f9-47dc-9f42-f97e8fb36956)

**Reply by rlaphoenix** [reply, marked as answer]
That shouldn't normally be necessary, but try `pip install toml`.

**Reply by djdancer** [reply]
Thanks, it's fixed my issue.
